### PR TITLE
doc: change 'bits type' to 'primitive type'

### PR DESCRIPTION
**Pull request by jrklasen**
improves readability for newbies

### CODE REVIEW

**Comment by fredrikekre** [comment]
I think the current documentation might be correct, see e.g. #21758 ?

File: doc/src/manual/types.md [hunk 1]
```diff
@@ -532,7 +532,7 @@ julia> struct Point{T}
 
 This declaration defines a new parametric type, `Point{T}`, holding two "coordinates" of type
 `T`. What, one may ask, is `T`? Well, that's precisely the point of parametric types: it can be
-any type at all (or a value of any bits type, actually, although here it's clearly used as a type).
+any type at all (or a value of any primitive type, actually, although here it's clearly used as a type).
```
Comment (yuyichao) [inline doc/src/manual/types.md]: This one shouldn't be changed.

**Review by yuyichao — CHANGES_REQUESTED** [review]

File: doc/src/manual/types.md (hunk 1 above)
Comment (jrklasen) [inline doc/src/manual/types.md]: thanks

**Comment by jrklasen** [comment]
The remaining two changes seem to be correct. The first is a description of a `primitive type`, the second is a reference to a later paragraph about primitive types

**Review by yuyichao — APPROVED** [review]

**Review by Sacha0 — APPROVED** [review]
Thanks @jrklasen! :)

**Comment by ararslan** [comment]
The Travis failures are unrelated (64-bit Linux is the Arnoldi thing, macOS is a timeout)

**Comment by jrklasen** [comment]
thanks

**Pull request merged**